### PR TITLE
add oname to store original file name

### PR DESCRIPTION
**Pull request by banv**
Relate to issues #21

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -60,6 +60,7 @@ function SocketIOFile(socket, options) {
 		var uploadTo = fileInfo.uploadTo || '';
 		var data = fileInfo.data || {};
 		var filename = fileInfo.name;
+		var oFileName = fileInfo.name;
```
Comment (rico345100) [inline index.js]: What about using originalFileName instead of oFileName? It could be confuse other people :)

**Comment by rico345100** [comment]
Thanks to the contribute! I have reviewed your code and seems ok, but need to edit the property name to prevent confuse other people.

File: index.js (hunk 1 above)
Comment (banv) [reply index.js]: OK, but how about emitObj.oname? 
should we change emitObj.oname to emitObj.originalName ?

Comment (banv) [reply index.js]: I'v change oFileName to originalFileName

Comment (rico345100) [reply index.js]: That would be great!

**Comment by rico345100** [comment]
Merged! Thanks for contribution.

**Comment by rico345100** [comment]
I will update documentation and publish to NPM registry after get back to home.

**Comment by banv** [comment]
Thank you, I'm waiting for published version

**Pull request merged**